App.js

```diff
@@ -1,16 +1,17 @@
 import React, {Component} from 'react';
-import {Container, Header, Title, Content, Footer, FooterTab, Button, Left, Right, Icon, Text} from 'native-base';
-import {Provider} from 'react-redux';
-import {createStore} from 'redux';
+import { Container, Header, Title, Content, Footer, FooterTab, Button, Left, Right, Icon, Text } from 'native-base';
```

Comment:
lets configure eslint to keep fromatting consistent

Reply:
Done

Reply:
did you push?